### ISSUE TITLE
MIDIMessageEvent data is not nullable

### ISSUE THREAD

**Issue by padenot** [WebAudio/web-midi-api#233]
STR in Chrome:

```js
> var event = new MIDIMessageEvent("something");
> console.log(event.data);
< null
```

`MIDIMessageEvent.data` is not nullable, so this should be an empty array. It's harmless to allow an empty `MIDIMessageEvent` but it's not particularly useful either. We can change the spec for compat instead of having Chrome fixed maybe.

**Comment by hoch** [comment]
```webidl
partial interface MIDIMessageEvent : Event {
  readonly attribute Uint8Array? data;
}
```
Does this look good? I agree that `null` is better.

**Comment by padenot** [comment]
> ```webidl
> dictionary MIDIMessageEventInit: EventInit {
>   Uint8Array? data;
> };
> ```

`?` is implied, the opposite being to specify `require` iirc.

**Comment by hoch** [comment]
Yes. Then we can lose the dictionary bit. I edited my comment above. Thanks!

**Comment by padenot** [comment]
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/webmidi/midi_message_event_init.idl;l=8-11 a related TODO in Chromium.

**Comment by padenot** [comment]
https://github.com/WebAudio/web-midi-api/issues/168 seems to be a dupe, I'll sort it out.

**Comment by mjwilson-google** [comment]
Fixed by #252
